src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -10,17 +10,71 @@ import { MetaOverheadStatistics } from './MetaOverheadStatistics';
 import { formatBytes } from '../../../utils/format-numbers';
 
 import type { Profile, ProfileMeta } from '../../../types/profile';
+import type { SymbolicationStatus } from '../../../types/state';
+import { typeof resymbolicateProfile } from '../../../actions/receive-profile';
+import { assertExhaustiveCheck } from '../../../utils/flow';
 
 import './MetaInfo.css';
 
 type Props = {
   profile: Profile,
+  symbolicationStatus: SymbolicationStatus,
+  resymbolicateProfile: resymbolicateProfile,
 };
 
 /**
  * This component formats the profile's meta information into a dropdown panel.
  */
 export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
+  /**
+   * This method provides information about the symbolication status, and a button
+   * to re-trigger symbolication.
+   */
+  renderSymbolication() {
+    const { profile, symbolicationStatus, resymbolicateProfile } = this.props;
+    const isSymbolicated = profile.meta.symbolicated;
+
+    switch (symbolicationStatus) {
+      case 'DONE':
+        return (
+          <>
+            <div className="metaInfoRow">
+              <span className="metaInfoLabel">Symbols:</span>
+              {isSymbolicated
+                ? 'Profile is symbolicated'
+                : 'Profile is not symbolicated'}
+            </div>
+            <div className="metaInfoRow">
+              <span className="metaInfoLabel"></span>
+              <button
+                onClick={resymbolicateProfile}
+                type="button"
+                className="photon-button photon-button-micro"
+              >
```

Comment:
I was a bit hesitant about the button size, but tried the other options like 100% width or 100% of the right section. They didn't look better 🤷‍♂